client-sdk/ts-web/core/src/types.ts

```diff
@@ -96,6 +160,16 @@ export interface CommonConsensusInfo {
     addresses: CommonConsensusAddress[];
 }
 
+/**
+ * Constraints are the Intel SGX TEE constraints.
+ */
+export interface CommonConstraints {
```

Comment:
This is originally under `sgx.Constraints` so maybe `SGXConstraints`?

Reply:
ya there's definitely some weirdness with this being a flat namespace and the Go codebase being hierarchical. currently there's a `Common` type name prefix and a `common.ts` with almost everything from there mixed together: node, entity, sgx, etc. these are ones that are mostly _type definitions_ and _constants_. the exceptions are modules that are mostly _helpers_: common/cbor -> `misc.ts` and common/crypto/... -> their own file and prefix, such as `signature.ts` and `hash.ts`. PVSS is on the fence.

Reply:
let's change `CommonConstraints` to `SGXConstraints` and similar. `CommonNode` will be `Node`. `SignatureSignature` will become `Signature`. but keep helpers and constants in `common.ts`.